tdrs-backend/tdpservice/parsers/validators.py

```diff
@@ -186,99 +191,108 @@ def sumIsLargerFunc(value):
 def matches(option):
     """Validate that value is equal to option."""
     return make_validator(
-        lambda value: value == option, lambda value: f"{value} does not match {option}."
+        lambda value: value == option,
+        lambda value, record_type, friendly_name, item_num: f"{record_type}: {value} does not match {option}."
```

Comment:
per discussion with @reitermb, it might be nice to have a way to override the `error_func` passed into `make_validator` here, per implementation of the validator

e.g., `matches('U')` for the update indicator might want something like `matches('U', lambda value: f"Update indicator custom error message")`

Reply:
If we do that here, should we consider doing that on all of the validators? I know we do this with other validators like `startsWith` and I wondered if we should consider adding that extra degree of freedom to all the validators? I will add it here now, just some food for thought!

Reply:
@jtimpe, Tagging just so it doesnt get lost.